cli/src/main/java/org/jboss/as/cli/impl/ReadlineConsole.java

```diff
@@ -609,6 +639,19 @@ private void printCollectedOutput() {
                     connection.write(l + Config.getLineSeparator());
                 }
             }
+            connection.write(ANSI.MAIN_BUFFER);
```

Comment:
This should also be called in a finally block. In case something goes wrong we should guarantee that the console is put in the nominal state.